tests/pages/settings/billing-page.ts

```diff
@@ -0,0 +1,25 @@
+import { hasClass, visitable, collection, text, attribute } from 'ember-cli-page-object';
+import createPage from 'codecrafters-frontend/tests/support/create-page';
+
+export default createPage({
+  membershipSection: {
+    scope: '[data-test-membership-section]',
+  },
+
+  paymentHistorySection: {
+    charges: collection('[data-test-payment-history-item]', {
+      amount: text('[data-test-amount]'),
+      failed: hasClass('text-red-600'),
+      refundText: text('[data-test-refund-text]'),
+    }),
+    emptyStateText: text('> div:last-child'),
```

Comment:
Do we really need `emptyStateText` here? Wouldn't just using `paymentHistorySection.text` (the default thing that ember-cli-page-object provides) work?